meta-mender-nxp/README.md

```diff
@@ -25,3 +25,48 @@ Setup environment
 Build
 
     $ bitbake core-image-base
+
+
+## Dependencies
+
+This layer depends on:
+
+```
+URI: https://github.com/Freescale/meta-freescale
+branch: warrior
+revision: HEAD
+```
+
+```
+URI: https://github.com/Freescale/meta-freescale-3rdparty
+branch: warrior
+revision: HEAD
+```
+
+```
+URI: https://github.com/Freescale/meta-freescale-distro
+branch: warrior
+revision: HEAD
+```
+
+```
+URI: https://github.com/boundarydevices/meta-boundary
+branch: master
+revision: 14d7a54464450e50bdd28b6cba505d1263bc1b41
+```
+
+```
+URI: http://cgit.openembedded.org/meta-openembedded/
+branch: warrior
+revision: HEAD
+```
```

Comment:
Do not need to list all dependecies, if they are "inherited" by other layers. eg. by adding:

```
URI: https://github.com/Freescale/meta-freescale-distro
branch: warrior
revision: HEAD
```

You are already saying that you have a dependency on meta-freescale and meta-oe since that layer depends on those.

So would shorten it to:

```
URI: https://github.com/boundarydevices/meta-boundary
branch: master
revision: 14d7a54464450e50bdd28b6cba505d1263bc1b41
```

```
URI: https://github.com/Freescale/meta-freescale-distro
branch: warrior
revision: HEAD
```
```
URI: https://github.com/Freescale/meta-freescale-3rdparty
branch: warrior
revision: HEAD
```

Reply:
Indeed, I will update, thanks !